include/ck/tensor_operation/gpu/device/impl/device_elementwise_normalization_impl.hpp

```diff
@@ -0,0 +1,591 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <iostream>
+#include <sstream>
+
+#include "ck/utility/math.hpp"
+#include "ck/utility/sequence.hpp"
+#include "ck/utility/reduction_operator.hpp"
+
+#include "ck/tensor_operation/gpu/device/device_elementwise_normalization.hpp"
+#include "ck/tensor_operation/gpu/device/device_reduce.hpp"
+#include "ck/tensor_operation/gpu/device/impl/device_reduce_common.hpp"
+#include "ck/tensor_operation/gpu/grid/gridwise_elementwise_layernorm_welford_variance.hpp"
+#include "ck/tensor_operation/gpu/grid/gridwise_set_buffer_value.hpp"
+#include "ck/host_utility/device_prop.hpp"
+#include "ck/host_utility/kernel_launch.hpp"
+
+// Y = layernorm( A + B, Beta, Gamma )
```

Comment:
You have more than one elementwise operation here.
I thought the naming of ElementwiseOperation and AccElementwiseOperation is confusing.
device_gemm is a good example to imitate.
I suggest to rewrite the comment like this
X = A +B
Y = Layernorm(X, beta, gamme)
Therefore, you can rename
ElementwiseOperation -> XElementwiseOperation
AccElementwiseOperation-> YElementwiseOperation